.travis.yml

```diff
@@ -5,7 +5,12 @@ before_script:
   - git submodule update --init --recursive
   - mkdir -p build
   - cd build
-  - cmake .. -DCMAKE_BUILD_TYPE=Release
+  - cmake .. -DCMAKE_BUILD_TYPE=Debug
 script:
   - cmake --build .
   - ctest -V
+after_success:
```

Comment:
Consider putting these commands in a shell script `.travis/upload-gcov-results` and running as 

```
after_success:
  - .travis/upload-gcov-results
```

Keeps `.travis.yml` organized when there are multiple `after_success` tasks.